irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -105,10 +105,9 @@ namespace iroha {
       try {
         pqxx::binarystring public_key(signatory.data(), signatory.size());
         transaction_.exec(
-            "INSERT INTO signatory(\n"
-            "            public_key)\n"
-            "    VALUES (" +
-            transaction_.quote(public_key) + ");");
+            "INSERT INTO signatory(public_key)\n"
```

Comment:
What's the purpose of newline char?

Reply:
@l4l Thanks your comment!
There is no special reason.
Is it something strange?

Reply:
Nope, just curious :)